pkg/machine/qemu/machine.go

```diff
@@ -407,7 +411,35 @@ func (v *MachineVM) Init(opts machine.InitOptions) (bool, error) {
 		WritePath: v.getIgnitionFile(),
 		UID:       v.UID,
 	}
+
 	err = machine.NewIgnitionFile(ign)
+	if err == nil {
+		// write to known_hosts
+		hd := homedir.Get()
+		known := filepath.Join(hd, ".ssh", "known_hosts")
+		f, err := os.OpenFile(known, os.O_APPEND|os.O_WRONLY, 0600)
+		if err != nil {
+			if os.IsNotExist(err) {
+				logrus.Warn("podman machine will soon require a known_hosts file to function properly. Podman will create and mamage keys within ~/.ssh/known_hosts")
+				return true, nil
+			}
+			return false, err
+		}
+		// normalize the line, then add it to known_hosts
+		normalizedLine := knownhosts.Normalize("localhost:" + strconv.Itoa(v.Port))
+		s, err := commonSSH.PublicKey(v.IdentityPath, []byte(""))
+		if err != nil {
+			return false, fmt.Errorf("failed to read identity %q: %w", v.IdentityPath, err)
```

Comment:
Could you verify this is not a stutter?  IE If the IdentiyPath does not exist, does this error show the path twice?